api.php

```diff
@@ -662,13 +694,143 @@
+     */
+    $sanity = file_exists(__DIR__.'/tmp/sanity-lock');
+
+    $lastSanity = (int)$db->single("SELECT val FROM config WHERE cfg='sanity_last'");
+    $sanitySync = (bool)$db->single("SELECT val FROM config WHERE cfg='sanity_sync'");
+    api_echo(['sanity_running' => $sanity, 'last_sanity' => $lastSanity, 'sanity_sync' => $sanitySync]);
+} elseif ($q === 'node-info') {
+    /**
+     * @api            {get} /api.php?q=node-info  21. node-info
+     * @apiName        node-info
+     * @apiGroup       API
+     * @apiDescription Returns details about the node.
+     *
+     * @apiSuccess {object}  data A collection of data about the node.
+     * @apiSuccess {string} data.hostname The hostname of the node.
+     * @apiSuccess {string} data.version The current version of the node.
+     * @apiSuccess {string} data.dbversion The database schema version for the node.
+     * @apiSuccess {number} data.accounts The number of accounts known by the node.
+     * @apiSuccess {number} data.transactions The number of transactions known by the node.
+     * @apiSuccess {number} data.mempool The number of transactions in the mempool.
+     * @apiSuccess {number} data.masternodes The number of masternodes known by the node.
+     * @apiSuccess {number} data.peers The number of valid peers.
+     */
+    $dbVersion = $db->single("SELECT val FROM config WHERE cfg='dbversion'");
+    $hostname = $db->single("SELECT val FROM config WHERE cfg='hostname'");
+    $acc = $db->single("SELECT COUNT(1) FROM accounts");
+    $tr = $db->single("SELECT COUNT(1) FROM transactions");
+    $masternodes = $db->single("SELECT COUNT(1) FROM masternode");
+    $mempool = $db->single("SELECT COUNT(1) FROM mempool");
+    $peers = $db->single("SELECT COUNT(1) FROM peers WHERE blacklisted<UNIX_TIMESTAMP()");
+    api_echo([
+        'hostname'     => $hostname,
+        'version'      => VERSION,
+        'dbversion'    => $dbVersion,
+        'accounts'     => $acc,
+        'transactions' => $tr,
+        'mempool'      => $mempool,
+        'masternodes'  => $masternodes,
+        'peers'        => $peers
+    ]);
+} elseif ($q === 'checkAddress') {
+    /**
+     * @api            {get} /api.php?q=checkAddress  22. checkAddress
+     * @apiName        checkAddress
+     * @apiGroup       API
+     * @apiDescription Checks the validity of an address.
+     *
+     * @apiParam {string} account Account id / address
+     * @apiParam {string} [public_key] Public key
+     *
+     * @apiSuccess {boolean} data True if the address is valid, false otherwise.
+     */
+
+    $address=$data['account'];
+    $public_key=$data['public_key'];
+    $acc = new Account();
+    if (!$acc->valid($address)) {
+        api_err(false);
+    }
+
+    $dst_b = base58_decode($address);
+    if (strlen($dst_b) != 64) {
+        api_err(false);
+    }
+    if (!empty($public_key)) {
+        if($acc->get_address($public_key)!=$address){
```

Comment:
same comment as before. Line 776 the `strlen($dst_b) != 64` has spaces before and after the `!=`, but here in 780, you don't.